content/CORETutorials/Storage/Pools/_index.md

```diff
@@ -90,12 +90,18 @@ You can add or remove this after creating the pool.
 You can add or remove this after creating the pool.
 {{< /expand >}}
 {{< expand "Hot Spare" >}}
-**Hot Spare** sets up drives as reserved to insert into **Data** vdevs when an active drive fails.
-Hot spares are temporarily used as replacements for failed drives to prevent larger pool and data loss scenarios.
-
-When a failed drive is replaced with a new drive, the hot spare reverts to an inactive state and is available again as a hot spare.
-
-When the failed drive is only detached from the pool, the temporary hot spare is promoted to a full data vdev member and is no longer available as a hot spare.
+A **Hot Spare** vdev sets up drives as reserved to prevent larger pool and data loss scenarios
```

Comment:
I don't think we need to use bold here as it sounds/reads like a general discussion rather than talking about the UI in particular.
```suggestion
A hot spare vdev sets up drives as reserved to prevent larger pool and data loss scenarios
```